code/modules/power/solar.dm

```diff
@@ -155,12 +148,6 @@
 				if(prob(25) && broken())
 					new /obj/item/weapon/shard(src.loc)
 
-/obj/machinery/power/solar/blob_act()
-	if(prob(75))
-		broken()
-		src.density = 0

```

Comment:
Notable stupid shit found in blob_act()
Density to 0, but also see below comment.